internal/scheduler/scheduler.go

```diff
@@ -31,6 +31,12 @@ func (s *scheduler) Start(ctx context.Context, interval time.Duration) {
 		} else {
 			s.logger.Info().Msg("re-scheduling all completed syncs to run again")
 		}
+
+		if err := s.db.CleanQueueLogs(ctx, 30); err != nil {
+			s.logger.Err(err).Msg("encountered error cleaning queue logs")
+		} else {
+			s.logger.Info().Msg("successfully removed all queue sync logs")
```

Comment:
let's clarify this log message to say `successfully removed repo sync jobs older than X days` (the "all" here sounds a little scary)

Reply:
lol It does